backend/src/main/resources/config/config.yml

```diff
@@ -58,7 +58,6 @@ projects:
       password: OVERRIDE_IN_LOCAL_CONFIG
   - id: showcase.entitlementcard.app
     importUrl: https://example.com
-    pipelineName: BerechtigungskarteShowcase
```

Comment:
I think you forgot to remove `pipelineName: SozialpassKoblenz` in L49.

Reply:
you are right. thx